src/client/pages/OfferNew/Checkout/hooks.ts

```diff
@@ -45,3 +46,22 @@ export const useScrollLock = (
       window.removeEventListener('touchmove', listener)
     }
   }, [outerWrapperRef, visibilityState])
+
+export const useSsnError = (editQuoteResult: EditQuoteMutationResult) => {
+  const [ssnBackendError, setSsnBackendError] = useState('')
```

Comment:
what about using `null` for the no-error state?

Reply:
True! I fix! 🙌